static/files/sitemap.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="utf-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+  <url>
+    <loc>https://cn.ubuntu.com/</loc>
+    <changefreq>monthly</changefreq>
```

Comment:
If you are basing these off of the u.com sitemaps, they use 'loc' and 'lastmod' for the [/static/files/sitemap.xml](https://ubuntu.com/static/files/sitemap.xml)

Reply:
Sorry, should've clarified, this one I based off of c.com as I didn't know the time of changes for any given page. Having read through some articles though it sounds like google indexes more efficiently with `lastmod` than with `changefreq`. I've updated but omitted the time as it turns out [it isn't required](https://www.sitemaps.org/protocol.html#xmlTagDefinitions)